hydrus/parser/openapi_parser.py

```diff
@@ -37,21 +37,113 @@ def check_if_collection(schema_block: Dict[str, Any]) -> str:
     :param schema_block: child of response object where schema is defined for return variables
     :return: collection (string)
     """
-    print(schema_block)
     try:
         type = schema_block["type"]
         print("type is " + type)
         if type == "array":
             collection = "true"
         else:
-            # TODO here in type we will get object,string,integer etc
             collection = type
     except KeyError:
         collection = "false"
     return collection
 
 
-def get_class_details(class_location: List[str], doc: Dict["str", Any], classAndClassDefinition: Dict["str",HydraClass],
+def parse_prop(
+        prop:str,
+        properties:Dict["str",Any],
+        definitionSet: Set["str"],
+        doc: Dict["str",Any],
+        classAndClassDefinition:Dict["str",HydraClass]) -> str:
+    """
+
```

Comment:
Some Docstrings have `descriptions` but lack `params` other have `params` but lack proper `descriptions`.
Fix this